lib/Dialect/OM/OMOps.cpp

```diff
@@ -535,6 +535,15 @@ PathCreateOp::verifySymbolUses(SymbolTableCollection &symbolTable) {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// IntegerAddOp
+//===----------------------------------------------------------------------===//
+
+FailureOr<llvm::APSInt>
+IntegerAddOp::evaluateIntegerOperation(llvm::APSInt lhs, llvm::APSInt rhs) {
```

Comment:
```suggestion
IntegerAddOp::evaluateIntegerOperation(const llvm::APSInt &lhs, const llvm::APSInt &rhs) {
```

Reply:
Ah thanks for the suggestion, that's a good point. I don't know how expensive APSInt is to copy, but no reason this can't be const ref.

Reply:
I see.. clang-tidy is also enforcing this. Since this is an interface method, I'll send a patch to main to enforce this in the interface definition itself as well.